pkg/arvo/sys/zuse.hoon

```diff
@@ -196,7 +196,7 @@
       [%crud p=@tas q=(list tank)]
       ::  boot completed (XX legacy)
       ::
-      [%init p=ship]
+      [%init ~]
```

Comment:
Does this mean the `XX legacy` qualifier can go?

Reply:
`XX somewhat legacy` ...